wagn/lib/wagn/generators/wagn/templates/config/routes.erb

```diff
@@ -1,8 +1,6 @@
 # -*- encoding : utf-8 -*-
 
-require 'decko/engine'
-
-Rails.application.routes.draw do
+Wagn.application.routes.draw do
```

Comment:
I don't think we need to mention an application there. See this part from http://guides.rubyonrails.org/engines.html
````
Blorgh::Engine.routes.draw do
end
```